### DISCUSSION TITLE
Cinnamon API reference and tutorials

### DISCUSSION THREAD

**Discussion by ghost** [linuxmint/discussions#710] [Tutorials]
These are not very exposed, so posting here

**GJS (cairo, Gtk, St and more)**: https://gjs-docs.gnome.org

**Cinnamon GJS**: run `sudo apt install cinnamon-doc devhelp` in the terminal. Then run `devhelp` for browsing the API reference, or open the app Devhelp from the menu.

**Comment by CyberAustin** [comment]
I don't know why this `sudo apt install cinnamon-doc devhelp` isn't at the top of every official page regarding Cinnamon dev. I've been pulling my hair out trying to make sense of the Gnome documentation that is linked everywhere.
